sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -235,6 +242,8 @@ clients_summary AS (
     environment.build.build_id AS env_build_id,
     environment.build.version AS env_build_version,
     environment.build.architecture AS env_build_arch,
+    environment.build.platform_version AS env_build_platform_version,
+    environment.build.xpcom_abi AS env_build_xpcom_abi,
```

Comment:
For most fields, the decision was to use `environment` instead of `application` as the source. However I  notice that the source in `new_profile` and `first_shutdown` pings for these fields is `application.xpcom_abi`. Worth a double check that this doesn't result would result in inconsistent data?

Reply:
I ran some queries looking for instances where they're different and didn't get any results at least for recent 2023 data